src/aiperf/metrics/list_metric_aggregation.py

```
@@ -0,0 +1,141 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Run-level aggregator for list-valued record metrics.
+
+Used by :class:`aiperf.post_processors.metric_results_processor.MetricResultsProcessor`
+when a ``MetricType.RECORD`` metric arrives with a list value (today only
+``inter_chunk_latency``, where each request contributes a list of inter-chunk
+gap durations). At 1 M-request ramp scale the exact storage —
+``records × (chunks-1) × 8 B`` — would dwarf the records-manager pod's
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)